### DISCUSSION TITLE
Home automation triggered by SS motion sensor seems to be based on delivery of the notification

### DISCUSSION THREAD

**Discussion by betabob** [homebridge-simplisafe3/homebridge-simplisafe3#314] [Q&A]
I noticed recently that an automation I've setup which is triggered by one of my motions sensors stopped working all of a sudden. But then I realized that it was pretty late at night and when my phone was in "sleep mode", so that all of my notifications are muted including the SS notification that motion was detected. As soon as I got out of sleep mode the automation worked correctly again. This leads me to believe that the automation is dependent not on the motion event itself but on the delivery of the _notification_ of that event. Can someone please verify if what I am saying is true? I've been able to work around this issue by allowing notifications from the SS app while in sleep mode, but it'd be great if I didn't have to. Thanks!

**Comment by shamoon** [comment]
I think if you do not have a 'hub' (i.e. apple tv, homepod) perhaps that may be the case, not sure. But if you do have a hub that is definitely not the case.

**Reply by betabob** [reply]
Hmm, I’ve actually got multiple hubs in my home. Anyway, I guess I can just live with the workaround, but based on what you say it does seem a bit odd.

**Reply by shamoon** [reply]
Do you have the automation setup on your phone only or as a home automation?

**Reply by betabob** [reply]
It’s a straight up home app automation. Doesn’t involve Siri Shortcuts or anything like that.

**Reply by shamoon** [reply]
Yea that doesnt make much sense to me. I just tested it to be sure too. Motion detected (camera not motion sensor but same same) --> turn on light. Set phone to focus, trigger sensor, light turns on.

I mean from a user perspective this would be insane, how would it choose one person in the home to 'control' this? What if a phone is off? And the SS notifications on your phone are totally unrelated, I have those notifications turned off and I only use HomeKit. You could even uninstall the SS app from your phone.

Have you tried deleting the automation and recreating it? Under Shortcuts > Automations it shows up as Home not Personal? Feel like theres a simple explanation somewhere...

**Reply by shamoon** [reply]
Do you have the motion sensor set to "secret alert" or "alarm" in every mode (e.g. off/home/away)?

**Reply by betabob** [reply]
Well, I just removed getting SS notifications on sleep mode, went into sleep mode, and now my automation works just fine 🤷‍♂️.  

And yeah, I had things setup properly per your questions above. Maybe just had some short term anomaly. Sorry to bother you with this, if it happens again and I can figure out what changed to have caused it I’ll let you know.
